### ISSUE TITLE
How to pass y_labels or scores to train model

### ISSUE THREAD

**Issue by deshwalmahesh** [idealo/image-quality-assessment#79]
I want to mimic your model but using TID2013 dataset. I have built the model but do not know how to train the model.  I use `ImageDataGenertor` of `Keras`

If I train the model using original scores with 1 output neuron, it'll be a regression problem  and it'll be bad.

If I train using `round(score)` with `sparse` input and 10 output neurons, it gives me `inf` loss

If I train using `categorical` or `raw`, same thing happens. How should I pass the training y_scores to the model?

**Comment by clennan** [comment]
Hi, what we did for TID2013 is to infer a probability distribution from the reported mean score. Then we passed the inferred distribution as labels throughout training with an EMD loss function. I also experimented with a regression approach using just the mean score as you mentioned which worked quite well.

**Comment by C-Aniruddh** [comment]
@clennan If I go with the regression approach and reduce the number of output neurons to 1, should I go with the `linear` activation function? Which function did you use in your approach?
